### ISSUE TITLE
.

### ISSUE THREAD

**Issue by joellemachaalany** [joellemachaalany/ift215-joelle-machaalany#2]
Dr. i have a question that has nothing to do with the project.
Can we built an android application using netbeans and java?
If yes , how can we run it on android?

**Comment by eddyghabachlcu** [comment]
Yes you can do it using NetBeans and Java but using a specific NetBeans platform for Android development. 
I suggest you to use Eclipse IDE for Android development since Eclipse entered mobile development earlier and there is more resources there. You need to download Android SDK, Eclipse and an Emulator (also called Simulator). Refer to android website to download all required resources. You can easily create an application there, copy the compiled APK file to your mobile and run it. Also the Emulator is a virtual mobile on which you can run/test your app.

**Comment by joellemachaalany** [comment]
thank you!
i have one more question, coding on eclipse is the same as netbeans??

**Comment by eddyghabachlcu** [comment]
Yes, but coding for Android is not exactly the same as Java. It's based on Java's API but Google made some modifications on it. But same concept.

**Comment by joellemachaalany** [comment]
okay thank you
